kafka-ui-react-app/src/components/common/Input/Input.tsx

```diff
@@ -23,14 +23,28 @@ const Input: React.FC<InputProps> = ({
   ...rest
 }) => {
   const methods = useFormContext();
+
+  const inputNumberCheck = (key: string, code: string) => {
```

Comment:
i think this function can be located outside of the component , and transformed into a pure function since it returns a boolean it does not have any side effects.

That way the function will not be created every-time `Input` component re-renders.

Reply:
Thanks for your suggestion. I forgot to take this into consideration, and I've fixed this in the new commit.